Closure download error

Hi, 

https://github.com/openlayers/closure-util/blob/c6be0af6801e4a41620e9f0701edf2a088ce68ed/default-config.json#L2

Since the compiler_url is using https instead of http (since version 1.18.0) in the default-config.json file, we get the following error installing closure-util :

```
ERR! closure-util self signed certificate in certificate chain
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! closure-util@1.22.0 postinstall: `node ./bin/closure-util.js update` npm ERR! Exit status 1 npm ERR!
npm ERR! Failed at the closure-util@1.22.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

```
I can't figure out, why we got this error, but I suspect our corporate proxy to be part of the problem.

In the closure wiki they never refer to the https version only the http one, could it be also part of the problem ? 
https://github.com/google/closure-compiler/wiki/Binary-Downloads

Thanks,

--- a/default-config.json
+++ b/default-config.json
@@ -1,5 +1,5 @@
 {
-  "compiler_url": "https://dl.google.com/closure-compiler/compiler-20170910.zip",
-  "library_url": "https://github.com/google/closure-library/archive/v20170806.zip",
+  "compiler_url": "http://dl.google.com/closure-compiler/compiler-20170910.zip",
+  "library_url": "http://github.com/google/closure-library/archive/v20170806.zip",
   "log_level": "info"
 }

